src/components/Footer.tsx

```diff
@@ -4,8 +4,11 @@ import { BULogo } from '../assets/assets.index';
 
 export const MainFooter: React.FC = () => {
   return (
+    <div>
+      <div className={styles.Partners}>Partners</div>
     <div style={{ width: '100vw%' }}>
       <img src={BULogo} className={styles.FooterLogo} />
     </div>
+    </div>
```

Comment:
this is not linted correctly. The husky git hook should have linted the project. Can you check `git status` and see if there's any changes to be commited. If there is, the changes should be from husky that linted the code. Commit the changes and push it up the branch.

How husky works is that it will automatically lint the project before pushing changes to the branch. However, the linted changes will not be included in the commit that are being pushed up; therefore, the linted changes will need to be committed and pushed again.

Reply:
git status does not show any changes to be committed. I understand how Husky works but it did not show any changes.
I'll fix this and commit.